buildSrc/src/main/kotlin/com/jakewharton/rxbinding/project/ValidateBindingsTask.kt

```diff
@@ -95,14 +94,11 @@ open class ValidateBindingsTask : SourceTask() {
     val parameters = method.parameters
     val statements = method.body.stmts
 
-    Observable.zip(
-        Observable.from(parameters)
-            .filter { it.type is ReferenceType }
-            .map { it.id.name },
-        Observable.from(statements),
-        { s, stmt -> Pair(s, stmt.toString()) }
-    )
-        .subscribe({
+    parameters
+        .filter { it.type is ReferenceType }
+        .map { it.id.name }
+        .zip(statements, { param, stmt -> Pair(param, stmt) })
```

Comment:
It looks like you lost a `.toString()` on `stmt` here which is causing the failure.